### PR TITLE
VZ-6632 - Add verrazzano-managed=true label on managed cluster resources created by Verrazzano

### PR DESCRIPTION
**Pull request by desagar**
Provide a summary of the change and which issue (i.e. ticket) is fixed.
If there are any dependencies, for example on a PR in another repository, list those.

### CODE REVIEW

**Review by mgianatagh — APPROVED** [review]

File: tests/e2e/pkg/keycloak.go [hunk 1]
```diff
@@ -216,7 +217,9 @@ func (c *KeycloakRESTClient) DeleteUser(userRealm string, userID string) error {
 		return fmt.Errorf("invalid response")
 	}
 	defer response.Body.Close()
-	if response.StatusCode != 200 {
+	// all responses in the 200 range are acceptable
+	// in practice, this call returns 204 (No Content) for success
+	if response.StatusCode < 200 || response.StatusCode >= 300 {
```
Comment (fredtibbitts) [inline tests/e2e/pkg/keycloak.go:222]: Are these keycloak.go changes supposed to be included in this PR? Just making sure this is intentional.

**Review by fredtibbitts — APPROVED** [review]
This looks fine to me. The only question I have is if we should be using an existing label, like "app.kubernetes.io/created-by". If we're already using the verrazzano-managed label in other places then it's probably better to be consistent and just use that.

File: tests/e2e/pkg/keycloak.go (hunk 1 above)
Comment (desagar) [reply tests/e2e/pkg/keycloak.go:222]: they are not specifically related to this. I made this change in a recent PR which fixed some jaeger tests, but that was reverted by Anand, so I'm including this here.

**Comment by desagar** [comment]
> This looks fine to me. The only question I have is if we should be using an existing label, like "app.kubernetes.io/created-by". If we're already using the verrazzano-managed label in other places then it's probably better to be consistent and just use that.

We are using verrazzano-managed to label application namespaces right now, including ones created by VerrazzanoProjects. I figured I could extend its use to these resources as well. I'll merge this now and we can discuss potentially using a different label. I know there is some work planned for standardizing labels.

**Pull request merged**